src/taggle.js

```diff
@@ -727,6 +727,11 @@
             this.pasting = true;
         }
 
+        if (this._isConfirmKey(key) && key === COMMA && this.input.value === '') {
```

Comment:
Why is there a check for `key === COMMA` when `this._isConfirmKey(key)` should handle that case? If a user doesn't have `COMMA` as a confirm key, it should be allowed so we can probably remove `key === COMMA`. Thoughts?

Reply:
`isConfirmKey` could also match `TAB` or `ENTER` in which case we don't want to prevent default.